tests/unit/ceph/test_ceph_primary_handlers.py

```
@@ -0,0 +1,1061 @@
+
+# ---------------------------------------------------------------------------
+# Import the ceph primary storage module with lock passthrough
+# ---------------------------------------------------------------------------
+try:
+    # Ensure lock decorators are passthroughs before importing
+    import tests.conftest  # noqa: F401
+
+    from tests.conftest import passthrough_lock
+    lock_mod = cast(object, importlib.import_module("zstacklib.utils.lock"))
+
+    setattr(lock_mod, "lock", passthrough_lock)
```

Ruff (0.15.2)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)